exercises/microwave/.meta/solutions/microwave.rb

```diff
@@ -2,10 +2,11 @@ class Microwave
   attr_reader :minutes, :seconds
 
   def initialize(digits)
-    base = digits > 99 ? 100 : 60
-
-    @minutes = digits / base
-    @seconds = (digits - minutes * base) % base
+    @minutes, @seconds = digits.divmod(100)
+    if @seconds >= 60
+      @seconds -= 60
+      @minutes += 1
+    end
```

Comment:
possible to replace these four lines with these two lines;

```ruby
extra_minute, @seconds = @seconds.divmod(60)
@minutes += extra_minute
```

However, I find such a replacement to be deceptive about the possibilities. At this point in the code, seconds can only be in range `0..99`, which means the only possible numbers of overflow minutes resulting from that are either 0 or 1. Using `divmod(60)` seems to imply to the reader of this code that it could be anything else. I prefer not to deceive the reader.

Reply:
I like the explicitness of the code stating that.  I agree.